src/actions/setup-challenge.ts

```diff
@@ -84,11 +91,39 @@ export const setupChallenge = async (name: string, installLocation: string) => {
         console.log(chalk.green("Challenge setup completed successfully."));
         console.log("");
         console.log(chalk.cyan(`Now open this repository in your favorite code editor and look at the readme for instructions: ${targetDir}`));
-    } catch (error) {
-        console.error(chalk.red("An error occurred during challenge setup:"), error);
+    } catch (error: any) {
+        console.error(chalk.red("An error occurred during challenge setup:"), error.message);
```

Comment:
I changed this console log from `error` to just the `error.message` because I thought it would be nice to hide the stack trace from the user. What do you think @escottalexander